### ISSUE TITLE
Private repos not supported

### ISSUE THREAD

**Issue by jasonbarry** [Daniel15/BuildSize#20]
Apologies if this is the wrong forum to ask support questions like this. 

I'm wondering if my configuration is set up correctly — BuildSize isn't commenting on my PRs nor setting a build status. Not sure what I'm doing wrong.

- I've installed BuildSize as a GitHub app and have given it the proper permissions
- CircleCI is monitoring my repo and running whenever a PR is open
- Build artifacts are being uploaded to CircleCI

<img width="719" alt="screen shot 2017-10-21 at 10 55 40 am" src="https://user-images.githubusercontent.com/236451/31854511-62b7f448-b64f-11e7-8c59-d2b821da3609.png">

Is there a specific path I need to put the build artifacts? 
My repo is private, can BuildSize only work with public repos?

Thank you!

**Comment by Daniel15** [comment]
> My repo is private, can BuildSize only work with public repos?

Yeah, that's the case at the moment. Sorry it's not fully documented - I built most of BuildSize over a weekend and didn't quite finish the documentation, and recently I've been very busy and haven't had a chance to work on it 😞 

I need to add support for providing CircleCI access tokens, which would allow private repositories to be used.
